tools/v8-prof/linux-tick-processor

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+TEMP_SCRIPT_FILE="/tmp/node-tick-processor-input-script"
```

Comment:
Suggestion: make this e.g. `/tmp/node-tick-processor-input-script-$$` to reduce the possibility of conflict with concurrent invocation and set `umask 077` before creating the file.